src/quick-start.md

```diff
@@ -28,13 +28,30 @@ helm repo add kubewarden https://charts.kubewarden.io
 helm install --wait -n kubewarden --create-namespace kubewarden-crds kubewarden/kubewarden-crds
 
 helm install --wait -n kubewarden kubewarden-controller kubewarden/kubewarden-controller
+
+helm install --wait -n kubewarden kubewarden-defaults kubewarden/kubewarden-defaults
 ```
 
 The following charts should be installed inside the `kubewarden` namespace in your Kubernetes cluster:
 
 * `kubewarden-crds`, which will register the `ClusterAdmissionPolicy` and `PolicyServer` Custom Resource Definitions
 
-* `kubewarden-controller` with a default configuration, and which will create a `PolicyServer` resource named `default`. 
+* `kubewarden-controller`, which will install the Kubewarden controller
+
+* `kubewarden-defaults`, which will create a `PolicyServer` resource named `default`.
+
+
+> **WARNING:**
+> Since [`v0.4.0`](https://github.com/kubewarden/kubewarden-controller/releases/tag/v0.4.0), a PolicyServer resource named default will not be created using the `kubewarden-controller` chart.
+> There is another Helm chart called `kubewarden-defaults`, which now installs
+> the default policy server.
+>
+> This also means that if you are not using the latest version of the `kubewarden-controller` and are trying to upgrade,
+> your default policy server will not be upgraded or deleted. As a result, you might still run into
+> issues if you try to install the `kubewarden-defaults` with some conflicting information ,for example the same policy server name.
+> Therefore, to be able to take advantage of future upgrades in the `kubewarden-defaults` helm chart you need to remove the
+> existing PolicyServer resource created by the `kubewarden-controller` before installing the new chart. With this step, you ensure that you are able to update your policy server using Helm upgrades without running into
```

Comment:
It might be worth mentioning that removing the default `PolicyServer` will also remove all policies bounded to it.